toolchain/sem_ir/facet_type_info.h

```diff
@@ -103,27 +103,76 @@ constexpr FacetTypeInfo::RewriteConstraint
 struct IdentifiedFacetType {
   using RequiredInterface = SpecificInterface;
 
-  IdentifiedFacetType() : interface_id(InterfaceId::None), num_to_impl(0) {}
+  IdentifiedFacetType()
+      : interface_id_(InterfaceId::None), num_interface_to_impl_(0) {}
```

Comment:
I think it's preferred to initialize these where the fields are defined. Then we can use `=default` for the ctor definition.